winrm/certgen.go

```diff
@@ -0,0 +1,255 @@
+// Copyright 2016 hoenirvili
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Also, please remove the ASL notice, the project is covered by the ASL globally, there's no need for file specific license notices.